release/create-release-candidate-branch.sh

```diff
@@ -43,75 +46,107 @@ tag_operators() {
 		nix-shell --run 'make regenerate-nix'
 
 		update_code "$TEMP_RELEASE_FOLDER/${operator}"
-		#-----------------------------------------------------------
+
 		# ensure .j2 changes are resolved
-		#-----------------------------------------------------------
 		"$TEMP_RELEASE_FOLDER/${operator}"/scripts/docs_templating.sh
-		#-----------------------------------------------------------
+
 		# inserts a single line with tag and date
-		#-----------------------------------------------------------
 		update_changelog "$TEMP_RELEASE_FOLDER/${operator}"
 
 		git commit -sam "release $RELEASE_TAG"
-		git tag -sm "release $RELEASE_TAG" "$RELEASE_TAG"
 		push_branch
 	done < <(yq '... comments="" | .operators[] ' "$INITIAL_DIR"/release/config.yaml)
 }
 
-tag_repos() {
+rc_branch_repos() {
 	if [ "products" == "$WHAT" ] || [ "all" == "$WHAT" ]; then
-		tag_products
+		rc_branch_products
 	fi
 	if [ "operators" == "$WHAT" ] || [ "all" == "$WHAT" ]; then
-		tag_operators
+		rc_branch_operators
 	fi
 }
 
+check_tag_is_valid() {
+	git fetch --tags
+
+	# check tags: N.B. look for exact match
+	TAG_EXISTS=$(git tag --list | grep -E "$RELEASE_TAG$")
+	if [ -n "$TAG_EXISTS" ]; then
+		echo "Tag $RELEASE_TAG already exists!"
```

Comment:
We can redirect stdout to stderr here, as is conventional.

```suggestion
		>&2 echo "Tag $RELEASE_TAG already exists!"
```